src/openforms/contrib/referentielijsten/api/serializers.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import serializers
+
+from openforms.api.utils import mark_experimental
+
+
+@mark_experimental
+class ReferentielijstTabellenSerializer(serializers.Serializer):
+    code = serializers.CharField(
+        help_text=_("The unique code that identifies the table.")
+    )
+    naam = serializers.CharField(help_text=_("The name of the table."))
+    is_geldig = serializers.SerializerMethodField(
+        help_text=_("Indicates whether or not the table is expired.")
+    )
+
+    def get_is_geldig(self, attrs: dict) -> bool:
```

Comment:
can we use a more specific type hint for `attrs`? plain `dict` usually doesn't say anything